<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Predict</class>
 <widget class="QWidget" name="Predict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户流失预测</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>本页面根据航空公司客户特征进行客户流失情况预测</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_back">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>主界面</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_Kmeans">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>聚类分析</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_predict">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>客户流失预测</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,5,0,0,0">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>步骤一导入数据：导入干净的数据（如temp/data_clean.xls）并进行数据预处理得到temp/data_predict.xls</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="PB_openfile">
           <property name="text">
            <string>导入数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_clean">
           <property name="text">
            <string>数据预处理</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DataTableWidget" name="pandasTableWidget" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>步骤二选择模型方法：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_model">
           <item>
            <property name="text">
             <string>SVM</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GBDT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RF</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_train">
           <property name="text">
            <string>开始训练</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Acc：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_acc"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="LB_stat">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>STATE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTableWidget</class>
   <extends>QWidget</extends>
   <header>qtpandas.views.DataTableView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
